int32[3] sensor_reading
